lib/src/components/NxNavigationSidebar/NxNavigationSidebar.scss

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-width__open: 255px;
+$nx-sidebar-width__closed: 72px;
+$nx-sidebar-logo-height: 34px;
+$nx-sidebar-link-height: 40px;
+
+$nx-sidebar-text-color: $nx-white;
+$nx-sidebar-background-color: $nx-blue-1000;
+$nx-sidebar-active-color: $nx-indigo-400;
+$nx-sidebar-hover-color: $nx-indigo-600;
+$nx-sidebar-selected-color: $nx-indigo-800;
+$nx-sidebar-separator-color: $nx-grey-900;
+
+.nx-page-sidebar--navigation {
+  @include container-vertical;
+
+  background-color: $nx-sidebar-background-color;
+  color: $nx-sidebar-text-color;
+  display: flex;
+  flex: none;
+  flex-direction: column;
+  width: $nx-sidebar-width__open;
+  padding: $nx-spacing-l 0 $nx-spacing-xxl 0;
```

Comment:
Measuring from the top of the screen to the top of the hexagon, there's 20 more pixels in the impl than in the mockup.

I think it might be a good design exercise to create a mockup with the button hovered and longer text content.  I worry the spacing as it is now (in the mockup, not in your impl) will lead to collisions fairly easily.

Reply:
> Measuring from the top of the screen to the top of the hexagon, there's 20 more pixels in the impl than in the mockup.

The mockup is not pixel perfect — See my note in the PR description.

Reply:
> The mockup is not pixel perfect — See my note in the PR description.

Mockups are required to be correct as they are the source of truth for what it's supposed to look like.  If we come back to this in a month and notice it doesn't match the mockup we can't be sure if that's on purpose or not, unless we know that we kept the mockup up-to-date and correct (in which case we know deviations from it found later on are in fact bugs).

Reply:
> > The mockup is not pixel perfect — See my note in the PR description.
> 
> Mockups are required to be correct as they are the source of truth for what it's supposed to look like. If we come back to this in a month and notice it doesn't match the mockup we can't be sure if that's on purpose or not, unless we know that we kept the mockup up-to-date and correct (in which case we know deviations from it found later on are in fact bugs).

You're asking me to affect something over which I have no power. This is a conversation you should be having with @stephenim and the larger design org.

Reply:
> > The mockup is not pixel perfect — See my note in the PR description.
> 
> Mockups are required to be correct as they are the source of truth for what it's supposed to look like. If we come back to this in a month and notice it doesn't match the mockup we can't be sure if that's on purpose or not, unless we know that we kept the mockup up-to-date and correct (in which case we know deviations from it found later on are in fact bugs).

This should be fixed now that we have an updated mock: https://app.zeplin.io/project/5ed14ea1b07f00b42032c55f/screen/6048fdf6469e5d088d247c26

Reply:
> This should be fixed now that we have an updated mock: https://app.zeplin.io/project/5ed14ea1b07f00b42032c55f/screen/6048fdf6469e5d088d247c26

The mockup now shows 32px here while the impl has 36.

Reply:
> > This should be fixed now that we have an updated mock: https://app.zeplin.io/project/5ed14ea1b07f00b42032c55f/screen/6048fdf6469e5d088d247c26
> 
> The mockup now shows 32px here while the impl has 36.

Should be fixed in b5fa5a7e2571d2544f9168368a76feeb026243c7